plasmapy/physics/tests/test_distribution.py

```diff
@@ -635,7 +692,467 @@ def test_value_drift_units(self):
                                        units="units")
         errStr = (f"Distribution function should be 0.0 "
                   f"and not {distFunc}.")
-        assert np.isclose(distFunc.value, 
+        assert np.isclose(distFunc.value,
                           0.0,
                           rtol=1e-8,
-                          atol=0.0), errStr
\ No newline at end of file
+                          atol=0.0), errStr
+
+
+#%% kappa
+
+# test class for kappa_velocity_1D function:
+class Test_kappa_velocity_1D(object):
```

Comment:
As an idea, perhaps one day we could just have a `Plasma` or `PlasmaBlob` instance for this kind of test?

Reply:
I'm not sure how that would work, especially as tests should thoroughly check edge cases. What do you envision?